src/nft/pages/collection/index.tsx

```diff
@@ -61,75 +63,81 @@ const Collection = () => {
 
   return (
     <>
-      <Column width="full">
-        {contractAddress ? (
-          <>
-            {' '}
-            <Box width="full" height="160">
+      <Trace
+        page={PageName.NFT_COLLECTION_PAGE}
+        properties={{ collection_address: contractAddress }}
```

Comment:
can we add chain_id as an event property here? i know it's only on eth right now, but that might not be the case forever so want to make sure we start tracking now.